src/main/scala-2.11/markets/participants/strategies/OrderCancellationStrategy.scala

```diff
@@ -0,0 +1,12 @@
+package markets.participants.strategies
+
+import markets.orders.Order
+
+import scala.collection.mutable
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1705050649/issues/source?bid=2921361&fileBranchId=3046267#l5)